source/config/configSpec.py

```diff
@@ -221,6 +221,9 @@
 	reportFrames = boolean(default=true)
 	reportClickable = boolean(default=true)
 
+[paragraphNavigation]
+	paragraphStyle = option("application", "normal", "block", default="application")

```

Comment:
Instead of setting the default directly, we tend to use an extra option "auto" which maps to a default behaviour. That way in the future we can switch the defaults while users remain on the default "auto".


```suggestion
[paragraphNavigation]
	paragraphStyle = option("auto", "application", "normal", "block", default="auto")
```